sale_stock_invoice_lot_quantity/views/report_invoice.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <template
+        id="custom_stock_account_report_invoice_document"
+        inherit_id="stock_account.stock_account_report_invoice_document"
+        priority="100"
+    >
+        <xpath expr="//t[contains(@t-esc, 'quantity')]" position="replace">
+            <!-- Call _format_quantity instead of using <span t-field=""> to avoid misalignment caused by the different behaviors of the t tag and span tag. -->
+            <t t-esc="snln_line['lot_quantity']._format_quantity()" />
+        </xpath>
```

Comment:
@AungKoKoLin1997 This looks much like overengineering.  Have you considered assigning `display:none` style to the `td` tag and redefining the whole `td` tag, populating quantity and UoM from the lot.quantity record?

Reply:
@yostashiro Redefining the whole `td` tag for these fields will also cause misalignment with other `td`. We will need to redefine all `td` tags.I don't like this approach because I don't want to touch other parts that is not related with our intention.